exchanges/stream/buffer/buffer_test.go

```diff
@@ -47,7 +47,7 @@ func createSnapshot() (holder *Orderbook, asks, bids orderbook.Items, err error)
 
 	ch := make(chan interface{})
 	go func(<-chan interface{}) { // reader
-		for range ch {
+		for range ch { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting